src/app/components/list/list-action-button.component.html

```diff
@@ -7,6 +7,9 @@
     <p>
       Lists display data in a consistent, flexible way.
     </p>
+    <sky-alert alertType="warning">
+      We recommend the <a href="https://developer.blackbaud.com/skyux/components/data-entry-grid">data entry grid component</a> or the <a href="https://developer.blackbaud.com/skyux-data-manager/docs/data-manager">data manager component</a> instead of lists because they extend the functionality and improve the behavior. We will deprecate the list module in a future version of SKY UX.
```

Comment:
should we only reference data manager, since data entry grid does not provide the functionality that list does? or maybe data manager + the standalone view components (data entry grid, repeater, ???)

Reply:
If you look at the individual components under docs, some link to data manager, some link to data entry grid, and some link to repeater. The links are based on the user story that Kerry created for these deprecation notes. So it seems to me that we shouldn't exclude data entry grid here since we are pointing to it on a subsequent page. Maybe we should add a link to repeater though.

Reply:
I'd be happier with including Repeater than with excluding Data Entry Grid. Really, the intent is to inform that this entire Big-L "List" ecosystem is old news, and we want consumers to be constructing little-l "lists" with the other components.

Reply:
FYI that I added a suggestion to tweak the alert to include a link to repeater and to switch verbiage from using to not imply that the components must be used in combination. (If this is incorrect and they do need to be used in combination, we need to reword the alerts on the individual pages.)